pallets/task/src/tests.rs

```diff
@@ -41,6 +42,9 @@ fn create_new_task(){
 
 		// Ensure new task can be created with [signer, specification, budget, deadline]
 		assert_ok!(Task::create_task(Origin::signed(1), TITLE.to_vec(), vec, 7, get_deadline()));
+		assert_eq!(Balances::balance(&1), 993);
```

Comment:
We should keep the tests small. Instead of adding multiple asserts into existing tests, please create 2-3 new tests that are only testing for balances when:

- Balance is transferred to the escrow account successfully when the task is started.
- Balance is transferred to volunteer once the task is accepted.
- Possible errors that might be thrown. 

The tests you have here are good, they just need to be specified in their own unit tests.